fv/wireguard_test.go

```diff
@@ -512,18 +512,9 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ WireGuard-Supported", []api
 				return readPolicy(policy.Name, api.Allow)
 			}, "5s", "100ms").ShouldNot(HaveOccurred())
 
-			cc.OnFail = func(msg string) {
-				log.WithField("msg", msg).Info("Connectivity Failed after 30seconds")
-				log.Info("Checking connectivity using ping between wls0 and wls1")
-
-				err, errstr := wls[0].SendPacketsTo(wls[1].IP, 5, 100)
-				log.Infof("wls0 to wls1: err=%v, errstr=%s", err, errstr)
-				err, errstr = wls[1].SendPacketsTo(wls[0].IP, 5, 100)
-				log.Infof("wls1 to wls0: err=%v, errstr=%s", err, errstr)
-			}
```

Comment:
I presume this is a case of removing debugging that we no longer need?

Reply:
Yes, but also this code would mask a failing test. [Here](https://github.com/projectcalico/felix/blob/master/fv/connectivity/conncheck.go#L56) it describes how if `OnFail` is set, `gingko.Fail()` won't be called.
Looking back through the commit history, this code was added for to debug a failing test, and presumably never got removed.

Reply:
Good catch.